Test/Unit/Model/Api/ShippingMethodsTest.php

```diff
@@ -173,7 +173,12 @@ public function setUp()
             ->willReturnSelf();
     }
 
-    public function testGetShippingMethodsIfQuoteEmptyReturnException()
+    /**
+     * @test
+     * @expectedException
```

Comment:
You don't need both `@expectedException` and `$this->expectException`.
Perhaps better to just go with the `$this->expectException` format, since it's implemented correctly here (the annotations need to expected values)